contribute.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Standup",
+    "description": "Web app that logs daily standup updates",
+    "repository": {
+        "url": "https://github.com/mozilla/standup",
+        "license": "BSD-3-Clause",
+        "tests": "https://travis-ci.org/mozilla/standup/"
+    },
+    "participate": {
+        "home": "https://wiki.mozilla.org/Standup",
+        "docs": "https://github.com/mozilla/standup#readme",
```

Comment:
This is a tough one because I want to create a RTD site for Standup the product which will have things like API docs and such on it, but it won't have Mozilla Standup maintenance things on it.

What does "docs" entail? Is it the product docs or the project docs? The `contributejson.org` site doesn't describe what the fields mean.